plugins/plugin-webpack/plugin.js

```diff
@@ -340,7 +340,10 @@ module.exports = function plugin(config, args = {}) {
               exclude: [/\.js?$/, /\.json?$/, /\.css$/],
               use: [
                 {
-                  loader: require.resolve('file-loader'),
+                  // When using old assets loaders (i.e. file-loader/url-loader/raw-loader)
+                  // make sure to set 'javascript/auto' flag
+                  // https://webpack.js.org/guides/asset-modules/
+                  loader: 'asset/resource',
                   options: {
                     name: assetsOutputPattern,
```

Comment:
Are you sure this works? The custom asset name goes either into `module.output.assetModuleFilename` or into `rules[].generator`, [according to the documentation](https://webpack.js.org/guides/asset-modules/#custom-output-filename). I have been making the same changes locally and I am still trying to figure out whether everything works as before.

Reply:
interesting, I made the same change locally (on project that's using webpack) and it seemed to work. Will double check tomorrow

Reply:
You're right. The I got bitten by that earlier as well (test not checking non .js files)

Reply:
I've made a [PR](https://github.com/mcha-dev/snowpack/pull/1) against your repo with suggested changes.

Reply:
Just saw this, I reverted to add a `.css` stub test